CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+**Internal**
+
+- Make sure to scrub all the fields with PII. If the fields contain an object, the entire object will be removed. ([#1789](https://github.com/getsentry/relay/pull/1789))
```

Comment:
Please also add this line to `py/CHANGELOG.md`, as per https://github.com/getsentry/relay#instructions-for-changelogs.

Reply:
done in [3811a6e](https://github.com/getsentry/relay/pull/1789/commits/3811a6e55ca078b360616470883ccf3f792771d2)